src/libraries/Common/src/Interop/Windows/Advapi32/Interop.RegCreateKeyEx.cs

```diff
@@ -16,7 +16,7 @@ internal static partial class Advapi32
     {
         // Note: RegCreateKeyEx won't set the last error on failure - it returns
         // an error code if it fails.
-        [DllImport(Libraries.Advapi32, CharSet = CharSet.Unicode, BestFitMapping = false, EntryPoint = "RegCreateKeyExW")]
+        [DllImport(Libraries.Advapi32, CharSet = CharSet.Unicode, BestFitMapping = false, EntryPoint = "RegCreateKeyExW", ExactSpelling = true)]
         internal static extern int RegCreateKeyEx(
```

Comment:
Not for this PR, but we might want to subsequently rename all of the methods to use the exact entrypoint name; doing that would make it more clear at the call site exactly which method was being used, would remove the need to specify EntryPoint as well, and also fits with the goal of minimizing magic.  Not a big deal, though.